README.md

```diff
@@ -544,6 +544,10 @@ The result of the code is the following output.
 You have deleted the document.
 ```
 
+## Error handling
+
+To see sample code, how we suggest to handle errors, please check [Cloudant API docs](https://cloud.ibm.com/apidocs/cloudant?code=go#error-handling)
```

Comment:
```suggestion
To see sample code, how we suggest handling errors, please check [Cloudant API docs](https://cloud.ibm.com/apidocs/cloudant?code=go#error-handling)
```

Reply:
This can be improved to:
```suggestion
For sample code on handling errors, please see [Cloudant API docs](https://cloud.ibm.com/apidocs/cloudant?code=go#error-handling)
```